### DISCUSSION TITLE
News about Designs in progress?

### DISCUSSION THREAD

**Discussion by yoggiland** [GoWired/GoWired-Project#29] [General]
Hi GoWired,

Since I read about the designs in progress (list below) I have eagerly awaited news about pending release.

Would it be possible for you (GoWired project owner) to give me/us a status update (as I am holding of on getting my GoWired boards until the new ones are released)?

Thanks in advance

yoggi

****
 	
- Multiprotocol Gateway - redesigned, multiprotocol gateway (2x RS485, CAN, USB) with powerful ATSAMD21 processor, W5500 Ethernet controller and an e-ink display

- 8RelayDin Shield - 8x 5 A / 230 V AC SPDT relays; 8x digital inputs

- Multisensor - Sensor-laden module which can work as a shield for MCU or as an I2C extension

- 4RelayDin Shield - 4x 16A / 230 V AC relays; independent current sensor on every channel

- 2EOL Shield - 8x 2EOL input for security sensors

- I/O Shield - 8IN / 8OUT module for various tasks, including direct interfacing with security systems

**Comment by feanor-anglin** [comment]
@yoggiland hello and thanks for your interest. The current status is as follows:

- Multiprotocol Gateway - complicated. There even is a [pre-launch campaign page on crowd supply](https://www.crowdsupply.com/domatic/gowired-multiprotocol-gateway), but the project turned out more complicated than we expected and we're a little short in time to develop it right now. But we'll definitely want to finish it. Don't expect the campaign to start this year though.
- 8RelayDin Shield - we have a batch of prototypes in our office, software has been intensively developed lately and some boards are in test use.
- Multisensor - no progress.
- 4RelayDin - there are PCBs waiting to be soldered (this would require some funds though).
- 2EOL Shield - we managed to manufacture one or two prototypes, which were quite usable, but no progress since then.
- I/O Shield - same as 2EOL Shield.

I have to admit, the progress done on these designs is not too impressive. The main limiting factor is that we met with rather modest response from the community and therefore we are rather short with budget. To change this state of affairs we devoted ourselves to develop [GoWired Touch](https://github.com/GoWired/GoWired-Touch) last year which we considered more attractive and impressive for larger target group. The whole project went pretty smooth (there is even a test installation with almost 40 Touch modules + ~10 touch stair switches), but we are still looking for an opportunity to put it on the market. 

To sum up, we are still out there and we still do want to develop GoWired (a lot of software progress recently) and we do look for opportunities to speed up this process, but we have to deal with limited resources problem. The most important help almost anyone could give us is to make our project a little more famous (at least among one's friends or on the internet).
